guides/common/modules/proc_exporting-a-repository.adoc

```diff
@@ -0,0 +1,82 @@
+[id="Exporting_a_Repository_{context}"]
+= Exporting a Repository
+
+You can export the content of a repository in the Library environment of an organization from {ProjectServer}.
+You can use this archive file to create the same repository in another {ProjectServer} or in another {ProjectServer} organization.
+
+You can export the following content from {ProjectServer}:
+
+* RPM repositories
+* Kickstart repositories
+* Ansible repositories
+* file repositories
+
+You cannot export Docker content
+ifndef::satellite[]
+nor DEB repositories
+endif::[]
+from {ProjectServer}.
```

Comment:
@maximiliankolb The situation is that DEB repos are not supported at all. We have verified this with Pulp developers.

Reply:
Ok, thanks for checking!